src/syft/ast/util.py

```diff
@@ -1,14 +1,10 @@
 # -*- coding: utf-8 -*-
-"""
```

Comment:
Woah, this the definition of legacy. This can go away for sure

Reply:
Yeah!